### PR TITLE
CN-7520: fixed wallet grid layout

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/molecules/CoinGridCard/CoinGridCard.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ class CoinGridCard extends Component {
     return (
       <Animated.View style={this.animate()}>
         <Card size="half" padding={"12 0 0 0"} onPress={onCardPress}>
-          <View style={style.cardInnerView}>
+          <View style={[style.cardInnerView, { height: 90 }]}>
```
Comment (CrnogoracIvan) [inline app/components/molecules/CoinGridCard/CoinGridCard.js:117]: Add similar percentage.

File: app/components/molecules/CoinGridCard/CoinGridCard.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ class CoinGridCard extends Component {
   );
 
   coinCardFull = coin => (
-    <Fragment>
+    <View style={{ flex: 1 }}>
```
Comment (CrnogoracIvan) [inline app/components/molecules/CoinGridCard/CoinGridCard.js:71]: Inline?

File: app/components/molecules/CoinGridCard/CoinGridCard.styles.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { getThemedStyle } from "../../../utils/styles-util";
 
 const base = {
   container: {
-    flex: 1,
+    // flex: 1,
```
Comment (CrnogoracIvan) [inline app/components/molecules/CoinGridCard/CoinGridCard.styles.js:5]: Delete or use.

**Review by CrnogoracIvan — CHANGES_REQUESTED** [review]
